doc/Build_on_docker.md

```diff
@@ -10,9 +10,68 @@ This on MacOS Catalina (10.15.3) and Mojave(10.14).
 - `sh docker_build.sh`
 - Somewhere in your host OS, run `socat TCP-LISTEN:6000,reuseaddr,fork UNIX-CLIENT:\"$DISPLAY\"`
 - Add your host machine IP address to xterm with `xhost` command.
-- Run docker with `sh quisp_docker.sh`
+- Run docker with `sh docker_run.sh`
 - Run `omnetpp` command on the docker container
 
+---
+
+## Easier way (Beta)
+
+This is just a beta version. If it's not working properly, please tell us as an issue.
+If you can't build with this process, please refer latter part.
```

Comment:
This is just a beta version of a newer, more automated (and hopefully more robust) installation process.  If it doesn't work properly for you, please open an issue on Github and/or contact us via Slack. You can also skip down lower in this file for the older, manual installation instructions.